src/Compilers/VisualBasic/Portable/Emit/SynthesizedPrivateImplementationDetailsSharedConstructor.vb

```diff
@@ -0,0 +1,116 @@
+﻿' Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+Imports System.Collections.Generic
+Imports System.Collections.Immutable
+Imports System.Diagnostics
+Imports System.Linq
+Imports Microsoft.CodeAnalysis.CodeGen
+
+Namespace Microsoft.CodeAnalysis.VisualBasic.Symbols
+    Friend NotInheritable Class SynthesizedPrivateImplementationDetailsSharedConstructor
+        Inherits SynthesizedGlobalMethodBase
+
+        Private ReadOnly _containingModule As SourceModuleSymbol
+        Private ReadOnly _privateImplementationType As PrivateImplementationDetails
+        Private ReadOnly _voidType As TypeSymbol
+
+        Friend Sub New(
+            containingModule As SourceModuleSymbol,
+            privateImplementationType As PrivateImplementationDetails,
+            diagnostics As DiagnosticBag,
```

Comment:
Will remove. This was left over from using an incorrect base type that needed a diagnostic bag.